src/status/libstatus/core.nim

```diff
@@ -16,7 +16,13 @@ proc callPrivateRPC*(methodName: string, payload = %* []): string =
       "method": methodName,
       "params": %payload
     }
-    result = $libstatus.callPrivateRPC($inputJSON)
+    echo "calling json"
+    echo $inputJSON
+    let response = libstatus.callPrivateRPC($inputJSON)
+    result = $response
+    if parseJSON(result).hasKey("error"):
+      echo "rpc response error"
+      echo result
```

Comment:
Do you still need the echos? If so, it's better to use `debug` or another chronicles method